doc/src/docs/InputOutputReference/01c-InputOutputReference.md

```diff
@@ -15446,6 +15448,183 @@ This field reports the amount of radiant energy transferred from the baseboard t
 
+### ZoneHVAC:CoolingPanel:RadiantConvective:Water
+
+The radiant/convective cooling panel is a component of zone equipment.  Chilled water is supplied by the plant and circulated through the panel.  Through radiation and convective heat transfer, the panel absorbs energy contained within the zone to provide cooling.  Control of the panel is achieved using the methodology used for low temperature radiant systems—a piecewise linear control profile that varies the flow rate to the panel based on a control parameter defined by the user.  This is different than being controlled to meet any remaining load like the air-based systems.  However, due to the radiant nature of these devices, it is not possible to control precisely on a zone air-based load since this will result in overcooling at the next time step.  Despite this, in addition to the piecewise linear controls that have been used with low temperature radiant systems in the past, two new controls will be added for this simple model: one that varies convective output of the system to meet the remaining zone load and one that varies the total convective output of the system to meet the remaining zone load.  In these two zone load control types, the user must enter the typical zone thermostat object to replace the setpoint and throttling range input in this object.  In this component, control is accomplished by throttling the chilled water flow to the device.  Capacity rating information is required as input and is used to calculate a UA value of the unit. The unit is connected to a chilled water loop (demand side) with an inlet and outlet node. The input data dictionary definition of the unit is as follows.
+
+#### Field: Name
+
+A unique user assigned name for an instance of radiant cooling panel. Any reference to this unit by another object will use this name.
+
+#### Field: Available Schedule Name
+
+The name of the schedule (ref: Schedule) that denotes whether the radiant cooling panel can run during a given hour. A schedule value greater than 0 (usually 1 is used) indicates that the unit can be on during the hour. A value less than or equal to 0 (usually 0 is used) denotes that the unit must be off for the hour.
+
+#### Field: Inlet Node Name
+
+The name of the water inlet node for the radiant cooling panel.
+
+#### Field: Outlet Node Name
+
+The name of the water outlet node for the radiant cooling panel.
+
+#### Field: Rated Inlet Water Temperature
+
+This field is the inlet temperature to the panel under conditions used to rate the capacity of the panel.  This value along with the other rated conditions is used to calculate the overall heat transfer coefficient (U-Value) that will be used to characterize the heat transfer between the water being circulated through the panel and the air in the space.  The default value is 5°C (44°F).
+
+#### Field: Rated Space Temperature
```

Comment:
@Myoldmopar Sorry...thinking too much like an architecture person there and not an EnergyPlus developer.  I'll change that to ZONE...